automotive/src/main/java/au/com/shiftyjelly/pocketcasts/di/AutomotiveAppModule.kt

```diff
@@ -0,0 +1,29 @@
+package au.com.shiftyjelly.pocketcasts.di
+
+import au.com.shiftyjelly.pocketcasts.repositories.di.DownloadCallFactory
+import au.com.shiftyjelly.pocketcasts.repositories.di.DownloadOkHttpClient
+import au.com.shiftyjelly.pocketcasts.repositories.di.DownloadRequestBuilder
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import okhttp3.Call
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import javax.inject.Singleton
+
+@Module
+@InstallIn(SingletonComponent::class)
+object AutomotiveAppModule {
```

Comment:
This is a duplicate fo the `AppModule`. We could create a new module shared only by the phone and wear apps to avoid this duplication, but that felt like it would be overcomplicating things. Maybe later something like that may make sense (I kind of doubt it though).